doc/dev/crimson/crimson.rst

```diff
@@ -285,7 +282,7 @@ To use fio to test ``crimson-store-nbd``,
       ninja crimson-store-nbd
 
 #. Run the ``crimson-store-nbd`` server with a block device. Please specify
```

Comment:
s/Please//

We don't ever need to ask nicely. They've already come to our documentation for information. (inb4: I know that this "Please" isn't your work.)